### DISCUSSION TITLE
How to run it on Windows environment

### DISCUSSION THREAD

**Discussion by ababeel-ceo** [chartbrew/chartbrew#206] [Q&A]
Hi Guy ,
How can run it on windows environment. pls help me

**Comment by razvanilin** [comment]
You can set up everything on Windows, just follow the setup guide here: https://docs.chartbrew.com/#installation-setup

Make sure you you follow the instructions and download a bash terminal such as GitBash, cygwin or msys2, otherwise you won't be able to run the setup commands.

**Reply by ababeel-ceo** [reply]
Thanks @razvanilin It worked
